.openshift-ci/begin.sh

```diff
@@ -20,7 +20,7 @@ if [[ -z "${SHARED_DIR:-}" ]]; then
     exit 0 # not fatal but worth highlighting
 fi
 
-if [[ "${JOB_NAME:-}" =~ -ocp-4- ]]; then
+if [[ "${JOB_NAME:-}" =~ -ocp-(4|stable)- ]]; then
```

Comment:
🤦 sorry about that. I forgot we had this

Reply:
@BradLugo do we need to update the master branch as well?